tests/integration/reusable_values.py

```diff
@@ -18,7 +23,9 @@ async def _set_up_reusable_values():
     OFFER = await sign_and_reliable_submission_async(
         OfferCreate(
             account=WALLET.classic_address,
-            sequence=WALLET.sequence,
+            sequence=await get_next_valid_seq_number(
+                WALLET.classic_address, ASYNC_JSON_RPC_CLIENT
+            ),
```

Comment:
Isn't this autofilled anyways? So the sequence doesn't need to be filled in separately.
ditto for every other test file